.github/workflows/bot-command.yml

```
@@ -33,6 +33,27 @@ jobs:
+        with:
+          script: |
+            const body = context.payload.comment.body;
+            if (/^\s+\/bot/.test(body)) {
+              github.rest.issues.createComment({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                body: ":warning: **Bot command ignored**: The `/bot` command must appear at the very beginning of the comment (no leading blank lines or spaces). Please post a new comment with `/bot` as the first character."
+              });
+            }
     runs-on: ubuntu-latest
```

actionlint (1.7.12)

[error] key "runs-on" is duplicated in "Bot-command-leading-whitespace" job. previously defined at line:42,col:5

(syntax-check)